lib/enhancements/artefact.rb

```diff
@@ -10,7 +10,7 @@ def self.multipart_formats
   end
 
   def migrated_format?
-    %w(answer help_page).include?(kind)
+    %w(answer help_page local_transaction).include?(kind)
```

Comment:
heads up: this will conflict with changes in https://github.com/alphagov/publisher/pull/565